### PR TITLE
Config

### PR DESCRIPTION
**Pull request by kaszim**
This branch creates a class for handling configuration of various parts in the application. See #48 .

I am a bit unsure if cfg really should be a global variable. Some input would be appreciated!

### CODE REVIEW

**Comment by kaszim** [comment]
@svenanderzen Can you just check this fast? Made some small adjustments when getting Travis to accept the build.

File: main.py [hunk 1]
```diff
@@ -69,8 +73,13 @@ def export_streams_to_clipboard(self):
...
         clipboard.setText(text, mode=clipboard.Clipboard)
 
-    def add_new_stream(self, *args, stream_url=None, stream_quality="best"):
+    def add_new_stream(self, *args, stream_url=None, stream_quality=cfg[CONFIG_QUALITY]):
         """Adds a new player for the specified stream in the grid."""
+        stream_url, status = QtWidgets.QInputDialog.getText(self, "Stream input", "Enter the stream URL:")
+        if not status:
+            return
+
+        # TODO: Add streams here.
```
Comment (svenanderzen) [inline main.py]: This doesn't seem right? Is this line still in master or is it your IDE that had a saved copy of something? I mean the above should already be in master right? Or am I mistaking?

Comment (mictab) [inline main.py]: It does look like it's already implemented in master 🤔

Comment (kaszim) [inline main.py]: How the hell did that line get there? 😨

Comment (svenanderzen) [inline main.py]: Could it be that you merged against your local `master` branch which wasn't updated and not `origin/master`? @kaszim

Comment (mictab) [inline main.py]: Try a `git pull --rebase origin master`.

Comment (mictab) [inline main.py]: I'm pretty confused as to why Github doesn't yell "CONFLICT CONFLICT".

Comment (kaszim) [inline main.py]: @svenanderzen Thats probably it... 😅 🔫

Comment (svenanderzen) [inline main.py]: @mictab please, no rebases on a publicly pushed branch if not necessary, do a merge instead ❤️

Comment (kaszim) [inline main.py]: I guess I broke github 🎉

**Comment by kaszim** [comment]
The problem should be resolved now :ok_man:

**Pull request merged**